recipes/ribolands/meta.yaml

```diff
@@ -36,7 +34,6 @@ requirements:
     - matplotlib-base
     - future
     - crnsimulator
-    - treekin ==0.5.1
 
 test:
```

Comment:
If possible a more extensive test then --version would be awesome :-)

Reply:
@s-will : Please comment concerning the upper version bound on python and optionally adding a test.